analysis_options.yaml

```diff
@@ -1,5 +1,7 @@
 include: package:pedantic/analysis_options.1.8.0.yaml
 analyzer:
+  enable-experiment:
+    - non-nullable
```

Comment:
Does this enable the analyzer for all plugins?

Reply:
right